.env

```diff
@@ -0,0 +1,5 @@
+DB_USER=robotiger
```

Comment:
@sharonkass I recommend changing passwords as soon as you can and removing this file from git. As it is already here and public, it's immediately a vulnerability to the app. We can discuss ways around this tomorrow. But make sure that it's taken care of soon (especially the password changes here and adding .env to .gitignore)

Reply:
@JonathanWexler - Thanks. I noted this issue in the PR description. We can remove it from git (.gitignore) but for now I just included it to make it easier for the developers (so they don't have to maintain their own .env file or env vars). I thought we'd have at least some protection since the heroku login info is still protected. I can remove it though if you are very concerned.